out_command_test.go

```diff
@@ -585,5 +602,37 @@ var _ = Describe("Out Command", func() {
 				Ω(*release.TagName).Should(Equal("version-0.3.12"))
 			})
 		})
+
+		Context("with generate_release_notes set to false", func() {
+			BeforeEach(func() {
+				request.Params.GenerateReleaseNotes = false
+			})
+
+			It("creates a release on GitHub without autogenerated release notes", func() {
+				_, err := command.Run(sourcesDir, request)
+				Ω(err).ShouldNot(HaveOccurred())
+
+				Ω(githubClient.CreateReleaseCallCount()).Should(Equal(1))
+				release := githubClient.CreateReleaseArgsForCall(0)
+
+				Ω(release.GenerateReleaseNotes).Should(Equal(github.Bool(false)))
+			})
+		})
+
+		Context("with generate_release_notes set to true", func() {
+			BeforeEach(func() {
+				request.Params.GenerateReleaseNotes = true
+			})
+
+			It("creates a release on GitHub with autogenerated release notes", func() {
+				_, err := command.Run(sourcesDir, request)
+				Ω(err).ShouldNot(HaveOccurred())
+
+				Ω(githubClient.CreateReleaseCallCount()).Should(Equal(1))
+				release := githubClient.CreateReleaseArgsForCall(0)
+
+				Ω(release.GenerateReleaseNotes).Should(Equal(github.Bool(true)))
```

Comment:
Not sure if it is possible to also verify `Body` from the created release?

Reply:
It's possible to set `generate_release_notes` to `true` as well as setting a `body`, in which case the body is prepended to the generated release notes. I'll add a check to confirm that both are sent to the `CreateRelease` function, and rely on the contract provided by `go-github` to make sure that results in the correct parameters being supplied to the GitHub API. Does that work for you?